editor.planx.uk/src/@planx/components/FileUploadAndLabel/SelectMultiple.tsx

```diff
@@ -151,18 +165,62 @@ const renderOption: AutocompleteProps<
   false,
   "div"
 >["renderOption"] = (props, option, { selected }) => (
-  <ListItem {...props}>
-    <Checkbox
-      data-testid="select-checkbox"
-      checked={selected}
-      inputProps={{
-        "aria-label": option.name
-      }}
-    />
-    <ListItemText sx={{ ml: 2 }}>{option.name}</ListItemText>
-  </ListItem>
+  <OptionComponent option={option} selected={selected} {...props} />
 );
 
+interface OptionComponentProps extends React.HTMLAttributes<HTMLLIElement> {
+  option: Option;
+  selected: boolean;
+}
+
+const OptionComponent: React.FC<OptionComponentProps> = ({
+  option,
+  selected,
+  ...props
+}) => {
+  const ref = useRef<HTMLLIElement>(null);
+
+  // Using a custom listbox has a breaking effect on the native scroll used by MUI's Autocomplete
+  // Using a MutationObserver we can listen for MUI applying the focusVisible class
+  // We then use this to indicate that we need to scroll the focused element into view
```

Comment:
This feels like a pretty complex solution to a small UI problem. Tried a few alternatives first, but if there's anything obvious I'm missing here please let me know!